vllm/v1/request.py

```diff
@@ -43,6 +45,12 @@ def __init__(
         self.num_prompt_tokens = len(self.prompt_token_ids)
         self._output_token_ids: List[int] = []
         self._all_token_ids: List[int] = self.prompt_token_ids.copy()
+        self.max_logprobs = sampling_params.logprobs
```

Comment:
It is confusing that the arguments to this function are `max_logprobs` and `max_prompt_logprobs` given how they are used here.

I think the argument names should be `logprobs` and `prompt_logprobs` given how they are used here